docs/extend/plugins_volume.md

```diff
@@ -164,3 +164,52 @@ this point.
 
 Respond with a string error if an error occurred.
 
+
+### /VolumeDriver.Get
+
+**Request**:
+```
+{
+    "Name": "volume_name"
+}
+```
+
+Get the volume info.
+
+
+**Response**:
+```
+{
+  "Name": "volume_name",
+  "Mountpoint": "/path/to/directory/on/host",
+  Err: ""
+}
+```
+
+Respond with a string error if an error occurred.
```

Comment:
Plain string -- this is the same in the rest of the file, lmk if we should chagne this to be more clear.

Any non-200 response should be used to indicate a protocol error but **not**, for instance, using 404 for a resource that doesn't exist.